app/api/microservice.py

```diff
@@ -0,0 +1,63 @@
+from fastapi import APIRouter ,Depends,HTTPException
+from app.schemas import MicroserviceInDBBase , MicroserviceCreate , MicroserviceTeamScorecardBase ,MicroserviceCreateApi, MicroserviceScoreCardCreate
+from app.crud import CRUDMicroservice, CRUDMicroserviceTeamScorecard ,CRUDTeam, CRUDScoreCard, CRUDMicroserviceScoreCard
+from typing import List
+from app import dependencies 
+from pydantic import BaseModel, Field
+
+
+class Value(BaseModel):
+    value: int | bool | float | str = Field(..., title="Value", description="Value of the metric")
+
+router = APIRouter()
+
+@router.get("/", response_model=List[MicroserviceInDBBase])
+#@router.get("/Dashboard", response_model=List[MicroserviceInDBBase])
+def get_all_services(microServices: CRUDMicroservice = Depends(dependencies.getMicroservicesCrud)):
+    return microServices.getAllServicesByTeamName()
+
+@router.get("/{service_id}", response_model=MicroserviceInDBBase)
+async def getOne_service(service_id: int, microServices: CRUDMicroservice = Depends(dependencies.getMicroservicesCrud)):
+    service = microServices.getByServiceIdWithTeamName(service_id)
+    if service is None:
+        raise HTTPException(status_code=404, detail="Service not found")
+    return service
+
+@router.get("/scorecards/{service_id}", response_model=MicroserviceTeamScorecardBase)
```

Comment:
Please move this endpoint to a new file in the api folder called scorecards.py and do the necessary adjustments

Reply:
I believe this is a one of Microservice operation to return all service detail , so i will update the path name 
https://github.com/tactful-ai/microcentral/pull/43/files#diff-8dc9ecf2eabdfea0bed728acf1163c7da8af51a22d552d1e12c908852128143dR26